evap/evaluation/tools.py

```diff
@@ -70,3 +73,100 @@ def set_content_disposition(self, filename):
             self["Content-Disposition"] = f"attachment; filename=\"{filename}\""
         except UnicodeEncodeError:
             self["Content-Disposition"] = f"attachment; filename*=utf-8''{quote(filename)}"
+
+
+class ExcelExporter(ABC):
+    default_styles = {
+        "default":                  xlwt.Style.default_style,
+        "headline":                 xlwt.easyxf("font: bold on, height 400; alignment: horiz centre, vert centre, wrap on; borders: bottom medium", num_format_str="0.0"),
+        "bold":                     xlwt.easyxf("font: bold on"),
+        "italic":                   xlwt.easyxf("font: italic on"),
+        "border_left_right":        xlwt.easyxf("borders: left medium, right medium"),
+        "border_top_bottom_right":  xlwt.easyxf("borders: top medium, bottom medium, right medium"),
+        "border_top":               xlwt.easyxf("borders: top medium"),
+    }
+
+    add_default_sheet = True
+    default_sheet_name = "EvaP Export"
```

Comment:
You could do `default_sheet_name = None` and then below check for `if self.default_sheet_name is not None:`. It's one setting less and the unused (?) default value becomes a feature.